core/sql/arkcmp/CmpStatement.cpp

```diff
@@ -1072,7 +1082,8 @@ CmpStatement::process(const CmpMessageDDLwithStatus &statement)
 				 odbcProcess,
 				 noTextCache,
 				 aqrPrepare,
-				 standaloneQuery))
+				 standaloneQuery,
+				 doNotCachePlan))
```

Comment:
Have you fixed the code paths for both embedded compiling and separate compiler process compilation ?